docs/src/rules/no-fallthrough.md

```diff
@@ -201,6 +203,29 @@ switch(foo) {
 }
 ```
 
+### allowEmptyCase
+
+Examples for **correct** code for the `{ "allowEmptyCase": true }` option:
+
+```js
+/* eslint no-fallthrough: ["error", { "allowEmptyCase": true }] */
+
+switch(foo){
+    case 1:
+
+    case 2: doSomething();
+}
+
+switch(foo){
+    case 1:
+    /*
+    Put a message here 
+    */
+    case 2: doSomething();
+}
+
+```
+
 :::
```

Comment:
`::: correct` before this code block and `:::` after the previous code block are missing.